src/components/CollaboratorModal/components/MainSubmodal.tsx

```diff
@@ -0,0 +1,251 @@
+import {
+  ModalHeader,
+  ModalBody,
+  Grid,
+  GridItem,
+  Text,
+  Box,
+  Divider,
+  FormControl,
+  Input,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalProps,
+  useFormControlContext,
+  Skeleton,
+  Stack,
+} from "@chakra-ui/react"
+import {
+  IconButton,
+  ModalCloseButton,
+  FormErrorMessage,
+  FormLabel,
+  Button,
+} from "@opengovsg/design-system-react"
+import _ from "lodash"
+import { useEffect, useState } from "react"
+import { FormProvider, useForm } from "react-hook-form"
+import { BiTrash } from "react-icons/bi"
+import { useParams } from "react-router-dom"
+
+import { useLoginContext } from "contexts/LoginContext"
+
+import * as CollaboratorHooks from "hooks/collaboratorHooks"
+
+import { Collaborator } from "types/collaborators"
+import { MiddlewareError } from "types/error"
+import { DEFAULT_RETRY_MSG, useSuccessToast } from "utils"
+
+import { ACK_REQUIRED_ERROR_MESSAGE } from "../constants"
+
+import { AcknowledgementSubmodalContent } from "./AcknowledgementSubmodal"
+
+const LAST_LOGGED_IN_THRESHOLD_IN_DAYS = 60
+
+const numDaysAgo = (previousDateTime: string): number => {
+  const currDateTime = new Date(Date.now())
+  const prevDateTime = new Date(previousDateTime)
+  return Math.floor(
+    (currDateTime.getTime() - prevDateTime.getTime()) / (60 * 60 * 24 * 1000)
+  )
+}
+
+interface CollaboratorListProps {
+  onDelete: (user: Collaborator) => void
+}
+
+const CollaboratorListSection = ({ onDelete }: CollaboratorListProps) => {
+  const { email } = useLoginContext()
+  const { siteName } = useParams<{ siteName: string }>()
+  const {
+    data: collaborators,
+    isError,
+  } = CollaboratorHooks.useListCollaboratorsHook(siteName)
+  const { isDisabled } = useFormControlContext()
+
+  return (
+    <Box mt="40px">
```

Comment:
Should this be in `rem` instead? Same for the height/padding below.

Reply:
fixed!